CLAUDE.md

```
@@ -0,0 +1,74 @@
+npm run dev       # Start development server
+npm run build     # Production build
+npm run lint      # Run ESLint
+npm run format    # Check Prettier formatting
+npm run format:fix # Auto-fix Prettier formatting
+```
+
+No test suite is configured.
+
+## Required Environment Variables
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)